README.md

```diff
@@ -126,14 +128,15 @@ $ pip install ./graph_def_editor
    `test.out` at the root of the project.
 
 
-## TensorFlow version compatibility
+## TensorFlow versions compatibility
 
 GraphDef Editor is fully supported for TensorFlow versions 1.14.x and 1.15.x.
 For TensorFlow 2.x some transforms might not work.
 
-To execute tests for specific TensorFlow versions run the following command from the repository root:
+To execute tests for specific TensorFlow version run the following command from the repository root:
```

Comment:
Thanks for making corrections to the docs! 

Would you mind tweaking this line to say "...for *a* specific TensorFlow version..." instead of "...for specific TensorFlow version"?